src/main/java/com/esri/samples/portal/oauth/OAuthController.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2016 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.portal.oauth;
+
+import java.text.SimpleDateFormat;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.security.AuthenticationManager;
+import com.esri.arcgisruntime.security.OAuthConfiguration;
+
+public class OAuthController {
+
+  @FXML
```

Comment:
Feel like it looks better this way, but we can decide on that.
![fxml](https://cloud.githubusercontent.com/assets/14575538/21424284/b924024a-c839-11e6-9f1a-86f8a29f8e15.PNG)